### ISSUE TITLE
Can we set the backgrounds' mode?

### ISSUE THREAD

**Issue by Roy-Orbison** [Grabacr07/SylphyHorn#38]
I'd like to use *Tile* so they span my monitors. I can change this from display settings, but it reverts to *Fit* as soon as I switch desktops.

**Comment by Roy-Orbison** [comment]
Even if it only kept the mode set in Windows' settings, rather than managing it in SylphyHorn's settings, that'd be enough.

**Comment by sharpga** [comment]
I second this proposal. I want to be able to set Fill as the mode on changing the desktop. The only way I can force this is with a policy change which is only useful if you are not on a corporate network which forces the style on each login.

**Comment by redmondChris** [comment]
Agreed - I just want solid colors set to Fill as my backgrounds to differentiate, but have three screens, all with different aspect ratios. Always reverting to _Fit_, the best I can do is properly fill the background on just one of my monitors.
